### PR TITLE
GP made Optional Patient PDS

### PR DESCRIPTION
**Pull request by NaminderSoorma**
GP made Optional Patient PDS

### CODE REVIEW

File: StructureDefinition/NHSDigital-Patient-PDS.StructureDefinition.xml [hunk 1]
```diff
@@ -38,28 +46,13 @@
...
-      <min value="1" />
-    </element>
-    <element id="Patient.generalPractitioner:registeredGPPractice">
-      <path value="Patient.generalPractitioner" />
-      <sliceName value="registeredGPPractice" />
-      <min value="1" />
-      <max value="1" />
+    <element id="Patient.address.extension:addressKey.extension">
+      <path value="Patient.address.extension.extension" />
+      <min value="2" />
```
Comment (omarisgreat) [inline StructureDefinition/NHSDigital-Patient-PDS.StructureDefinition.xml]: Could we remove the constraint on addressKey entirely? There may be up to 2 for PDS there may also be none at all.

**Review by omarisgreat — CHANGES_REQUESTED** [review]
1 change requested, fine with the rest

**Review by NaminderSoorma — COMMENTED** [review]
Changes made to include EPS requirements for Patient addresskey.

**Review by omarisgreat — APPROVED** [review]
Looks good to me

**Pull request merged**